plop-templates/Component/Component.tsx.hbs

```diff
@@ -12,6 +12,10 @@ export interface Props {
 /**
  * BETA: This component is still a work in progress and is subject to change.
  *
+ * ```ts
+ * import { {{pascalCase name}} } from "@chanzuckerberg/eds";
```

Comment:
I added a space between the outer brackets because otherwise the brackets didn't show up in the newly created file at all.